tests/cc19/input.dat

```diff
@@ -33,7 +33,7 @@ ref456tensor = psi4.core.Matrix.from_list(        # TEST
 compare_values(refnre, hof.nuclear_repulsion_energy(), 6, "Nuclear Repulsion Energy") # TEST
 compare_values(refscf, wfn.variable("SCF TOTAL ENERGY"), 6, "SCF Energy") # TEST
 compare_values(refccsd, wfn.variable("CCSD TOTAL ENERGY"), 6, "CCSD Energy") # TEST
-compare_values(refoverlap, wfn.variable("Left-Right CCSD Eigenvector Overlap"), 5, "Left-Right CCSD Overlap") # TEST
+compare_values(refoverlap, wfn.variable("LEFT-RIGHT CCSD EIGENVECTOR OVERLAP"), 5, "Left-Right CCSD Overlap") # TEST
```

Comment:
was this needed? accessing should be case-insensitive https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/wavefunction.cc#L1325

Reply:
Not needed. This was a way for me to force-refresh the tests after a failure unrelated to this PR.

Reply:
got it, tks. adding/removing a blank line in tests/CMakeLists.txt also tricks it. you'd think CMake would be smarter :-)